hw/top_earlgrey/data/pins_cw341.xdc

```diff
@@ -0,0 +1,135 @@
+## Copyright lowRISC contributors.
+## Licensed under the Apache License, Version 2.0, see LICENSE for details.
+## SPDX-License-Identifier: Apache-2.0
+
+## ChipWhisperer CW341 "Bergen Board".
+##
+
+## Clock Signal
+set_property -dict { PACKAGE_PIN E22 IOSTANDARD LVCMOS18 } [get_ports { IO_CLK }]; # PLL_CLK2
+
+## Clock constraints
+## set via clocks.xdc
+
+## Power-on Reset
+set_property -dict { PACKAGE_PIN  G26 IOSTANDARD LVCMOS18 PULLTYPE PULLUP } [get_ports { POR_N }]; # Main PORN, requires jumper to SW0 [was USB_A14 (SAM3X)]
+
+## Preserve prim_prince modules and setup multi-cycle paths
+## These are no longer required, but kept here as a reference
+## set_property KEEP_HIERARCHY TRUE [get_cells top_earlgrey/u_flash_eflash/gen_flash_banks[*].i_core/u_scramble/u_cipher]
+## set_multicycle_path -setup 2 -through [get_pins -of_objects [get_cells top_earlgrey/u_flash_eflash/gen_flash_banks[*].i_core/u_scramble/u_cipher]]
+## set_multicycle_path -hold 1  -through [get_pins -of_objects [get_cells top_earlgrey/u_flash_eflash/gen_flash_banks[*].i_core/u_scramble/u_cipher]]
+
+#set_property CLOCK_DEDICATED_ROUTE FALSE [get_nets IO_SDCK_IBUF]; # SDCK clock to be ignored
+
+
+## IOA bank
+# UART (SAM3X)
+set_property -dict { PACKAGE_PIN AN26 IOSTANDARD LVCMOS18 } [get_ports { IOA0 }]; # SAM_TXD0 - OpenTitan UART2 RX
+set_property -dict { PACKAGE_PIN AK26 IOSTANDARD LVCMOS18 } [get_ports { IOA1 }]; # SAM_RXD0 - OpenTitan UART2 TX
+# GPIOs (PMOD1)
+set_property -dict { PACKAGE_PIN AN27  IOSTANDARD LVCMOS18 } [get_ports { IOA2 }]; # PMOD1_IO1 (OT CTS)
+set_property -dict { PACKAGE_PIN AP26  IOSTANDARD LVCMOS18 } [get_ports { IOA3 }]; # PMOD1_IO4 (OT RTS)
+# UART (PMOD1)
+set_property -dict { PACKAGE_PIN AP28  IOSTANDARD LVCMOS18 } [get_ports { IOA4 }]; # PMOD1_IO3 (OT RXD)
+set_property -dict { PACKAGE_PIN AM27  IOSTANDARD LVCMOS18 } [get_ports { IOA5 }]; # PMOD1_IO2 (OT TXD)
+# GPIO (PMOD1)
+set_property -dict { PACKAGE_PIN AP29  IOSTANDARD LVCMOS18 } [get_ports { IOA6 }]; # PMOD1_IO5 (INT)
+# I2C (PMOD1)
+set_property -dict { PACKAGE_PIN AN28  IOSTANDARD LVCMOS18 } [get_ports { IOA7 }]; # PMOD1_IO7 (SCL)
+set_property -dict { PACKAGE_PIN AP30  IOSTANDARD LVCMOS18 }  [get_ports { IOA8 }]; # PMOD1_IO8 (SDA)
+
+## IOB bank
+# Aux SPI host
+set_property -dict { PACKAGE_PIN AP8  IOSTANDARD LVCMOS18 } [get_ports { IOB0 }]; # USR_SPI1CLK
+set_property -dict { PACKAGE_PIN AN8  IOSTANDARD LVCMOS18 } [get_ports { IOB1 }]; # USR_SPI1DQ0
+set_property -dict { PACKAGE_PIN AP9  IOSTANDARD LVCMOS18 } [get_ports { IOB2 }]; # USR_SPI1DQ1
+set_property -dict { PACKAGE_PIN AL8  IOSTANDARD LVCMOS18 } [get_ports { IOB3 }]; # USR_SPI1CS
+# ChipWhisperer UART
+set_property -dict { PACKAGE_PIN AL10 IOSTANDARD LVCMOS18 } [get_ports { IOB4 }]; #J14 PIN 10 CWIO_IO1 - OpenTitan UART1 RX
+set_property -dict { PACKAGE_PIN AN9  IOSTANDARD LVCMOS18 } [get_ports { IOB5 }]; #J14 PIN 12 CWIO_IO2 - OpenTitan UART1 TX
+# GPIOs (DIP switches)
```

Comment:
I should probably fix up these annotations, since they may not completely reflect what is connected on the CW340.